services/api-server/src/simcore_service_api_server/api/routes/studies_jobs.py

```diff
@@ -191,23 +212,33 @@ async def start_study_job(
     webserver_api: Annotated[AuthSession, Depends(get_webserver_session)],
     director2_api: Annotated[DirectorV2Api, Depends(get_api_client(DirectorV2Api))],
     cluster_id: ClusterID | None = None,
-) -> JobStatus:
+) -> JobStatus | JSONResponse:
     job_name = _compose_job_resource_name(study_id, job_id)
     with log_context(_logger, logging.DEBUG, f"Starting Job '{job_name}'"):
-        await start_project(
-            request=request,
-            job_id=job_id,
-            expected_job_name=job_name,
-            webserver_api=webserver_api,
-            cluster_id=cluster_id,
-        )
-        job_status: JobStatus = await inspect_study_job(
+        try:
+            await start_project(
+                request=request,
+                job_id=job_id,
+                expected_job_name=job_name,
+                webserver_api=webserver_api,
+                cluster_id=cluster_id,
+            )
+        except ProjectAlreadyStartedError:
+            job_status: JobStatus = await inspect_study_job(
+                study_id=study_id,
+                job_id=job_id,
+                user_id=user_id,
+                director2_api=director2_api,
+            )
+            return JSONResponse(
+                content=jsonable_encoder(job_status), status_code=status.HTTP_200_OK
```

Comment:
I still find very confusing that 202 is the successful and that 200 is , as you call it, "the conflict".  Moreover, before 200 was "the successful" result.

THOUGHT: I do not see the problem of using docker approach to indicate "already started". Look how clear is defined https://docs.docker.com/engine/api/v1.45/#tag/Container/operation/ContainerStart
<img width="281" alt="image" src="https://github.com/ITISFoundation/osparc-simcore/assets/32402063/f2210b80-5e33-409e-93b2-2ed50adbc655">

https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/304 

The HTTP 304 Not Modified client redirection response code indicates that there is **no need to retransmit the requested resources.**

Reply:
Note this comment in the link you send
![image](https://github.com/ITISFoundation/osparc-simcore/assets/126242332/c8a22762-b31b-4dc8-80a3-8f474516dfae)
I would like to return the jobstatus also when the job was already started, that way the end user will not notice that the computation was already started.

Reply:
yes, yes... I did not read carefully your description :-) I get it.

Reply:
> I still find very confusing that 202 is the successful and that 200 is , as you call it, "the conflict". Moreover, before 200 was "the successful" result.

OK, I get this. I am also not completely satisfied with this approach. But I don't see a better way to do it. If you see it, please let me know.